### PR TITLE
Fixes xVMHyperV snapshot bug

### LINKED ISSUE #28: xVMHyperV fails if VM has snapshot
When a VM has a snapshot on the underlying VHD, the xHyperV resource fails due to the the VHD now pointing to the snapshot AVHDX file.Should we check for parent/root VHD file in the Test-TargetResource method?

### PR DESCRIPTION
**Pull request by iainbrighton**
* Fixes #28 
* Adds xVMHyperV localization support
* Updates readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ Please see the Examples section for more details.
 * MSFT_xVMHyperV: Changed the MACAddress parameter to string[] to support assigning multiple MAC addresses to virtual machines.
 * MSFT_xVMHyperV: Added enabling of Guest Service Interface.
 * MSFT_xVMSwitch: Added the BandwidthReservationMode parameter which specifies how minimum bandwidth is to be configured on a virtual switch
+* MSFT_xVMHyperV: Fixed bug causing Test-TargetResource to fail when VM had snapshots.
```
Comment (TravisEz13) [inline README.md]: you added your summary of changes in the current release not unreleased

**Comment by TravisEz13** [comment]
only one issue that needs to be addressed that I found.  Looks pretty good.

File: README.md (hunk 1 above)
Comment (iainbrighton) [inline README.md]: Not sure how I did that! Fixed :)

**Comment by TravisEz13** [comment]
Thanks for your contribution!

**Pull request merged**